config/reverse_proxy/nginx.conf

```diff
@@ -0,0 +1,65 @@
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    upstream relay {
+        server sentry_relay.sentry:3000;
+    }
+
+    upstream sentry {
+        server host.docker.internal:8888;
+    }
+
+    server {
+        listen 80;
+        resolver 127.0.0.11 ipv6=off;
+
+        location /api/store/ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/store/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/minidump/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/ureal/$ {
```

Comment:
```suggestion
        location ~ ^/api/\d+/unreal/$ {
```

Reply:
Strictly speaking, this is not the full URL, but since this is going to be the only unreal URL, this is fine.

Reply:
Updated to include `\w+`